client/package.json

```diff
@@ -22,8 +22,10 @@
     "@angular/router": "^16.2.0",
     "@ng-bootstrap/ng-bootstrap": "^15.1.2",
     "@popperjs/core": "^2.11.6",
+    "angular2-promise-buttons": "^6.0.0",
     "bootstrap": "^5.2.3",
     "rxjs": "~7.8.0",
+    "sweetalert2": "^11.10.3",
```

Comment:
Nice commit message, thanks for mentioning that you installed these and why! Very helpful